src/rules/function-space-after/index.js

```diff
@@ -0,0 +1,45 @@
+import {
+  ruleMessages,
+  isWhitespace
+} from "../../utils"
+
+export const ruleName = "function-space-after"
+
+export const messages = ruleMessages(ruleName, {
+  expected: "Expected single space after function's \")\"",
+  rejected: "Unexpected space after function's \")\"",
+})
+
+const acceptableNextChars = [ " ", ")", undefined ]
+
+/**
+ * @param {"always"|"never"} expectation
+ */
+export default function (expectation) {
+  return function (css, result) {
+    css.eachDecl(function (decl) {
+      if (decl.prop === "content") { return }
+
+      const value = decl.value
+
+      for (let i = 0, l = value.length; i < l; i++) {
+        if (value[i] === ")") {
```

Comment:
Did you consider using [balanced-match](https://www.npmjs.com/package/balanced-match) for this kind of test ?